Updated Terminal.tsx and Terminal.scss

lawtlee opened the pull request:
Added a copy button for code snippets.

<!--
    Your PR title can should describe what feature was added/changed
-->

## Summary

Closes #164 

<!-- Enumerate changes you made and why you made them in bullet form-->
<!-- list any new dependencies required for this change -->
- Added copy button to code snippets
- Added two dependencies:
  - react-copy-to-clipboard
  - react-icons

## Screenshots

<!--
    Add Screenshots of the feature in play, terminal pastes, etc. as necessary
-->

https://github.com/uclaacm/parcel-pointers/assets/104863284/4b4e002f-2222-4cbd-b055-b01f55e42141

lawtlee committed:
Updated Terminal.tsx and Terminal.scss

Added a copy button for code snippets.

Dankco commented on src/styles/Terminal/Terminal.scss:
```diff
@@ -1,5 +1,16 @@
+  right: 14px;
+  top: 14px;
+
+  &:hover {
```
Can you make the hover state also set the cursor to be a pointer? Otherwise, looks good!

lawtlee replied:
https://github.com/uclaacm/parcel-pointers/assets/104863284/ab7d1e01-b9b1-4d58-8e01-5a3e14d085b8

Dankco reviewed (commented):
I didn't realize that alyssa actually put a guide to this on the ticket- always thought it would require some research, but good job!

lawtlee committed:
Updated Terminal.tsx, Terminal.scss

Added a little alert at the top of the screen
to alert you when you copy the code.

Dankco reviewed (approved):
Looks good!
